pom.xml

```diff
@@ -30,7 +30,7 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>mod-custom-fields</artifactId>
-      <version>1.3.0</version>
+      <version>1.3.1-SNAPSHOT</version>
```

Comment:
We don't usually add snapshot versions

Reply:
I agree, this was at the suggestion of the custom field folks, and defiantly was not working without it. Should this update hold until custom-fields is released?

Reply:
I'm fine to leave snapshot if it is updated to released version once available. The module can not be released util we have the lib released, I think.

Reply:
@jeremythuff Please can you raise an issue for updating this dependency to a release version and add it as a blocker for releasing mod-users.